src/diamonds/nayms/libs/LibMarket.sol

```diff
@@ -266,15 +265,15 @@ library LibMarket {
             // If both are internal tokens no commissions are paid
```

Comment:
Is this comment now redundant?

Reply:
yes, I believe so, I don't think we support such case at the moment, I will remove it